.gitignore

```diff
@@ -27,3 +27,8 @@ bundle/
 .vagrant/
 phantomjsdriver.log
 selenium-debug.log
+src/sandstorm/*.capnp.c++
+src/sandstorm/*.capnp.h
```

Comment:
Hmm that's weird, they must've ended up there for me from some other `capnp compile` gone rogue.